data.sql

```diff
@@ -212,8 +212,8 @@ INSERT INTO editii VALUES(185,7,'revista',NULL,183,2013,2,'','14.99',100,'DVD (1
 INSERT INTO editii VALUES(186,7,'revista',NULL,184,2013,3,'','14.99',100,'DVD (1)','Codename: Panzers - Cold War','KiMO','','',100,NULL,'GQ - recrop','Cristan');
 INSERT INTO editii VALUES(187,7,'revista',NULL,185,2013,4,'','15.98',100,'DVD (1)','Just Cause','KiMO','','',100,NULL,'GQ - recrop','Cristan');
 INSERT INTO editii VALUES(188,7,'revista',NULL,186,2013,5,'2013.06','15.98',100,'DVD (1)','Deus Ex: Human Revolution','KiMO','','',100,NULL,'GQ - recrop','Cristan');
-INSERT INTO editii VALUES(189,7,'revista',NULL,187,2013,7,'','15.98',100,'DVD (1)','Supreme Commander, Hacker Evolution Duality','KiMO','','',100,NULL,'GQ - recrop','Cristan');
-INSERT INTO editii VALUES(190,7,'revista',NULL,188,2013,8,'','15.98',100,'DVD (1)','Kane & Lynch','KiMO','','',100,NULL,'GQ - recrop','Cristan');
+INSERT INTO editii VALUES(189,7,'revista',NULL,187,2013,7,'','15.98',100,'DVD (1)','Supreme Commander 2, Hacker Evolution Duality','KiMO','','',100,NULL,'GQ - recrop','Cristan');
+INSERT INTO editii VALUES(190,7,'revista',NULL,188,2013,8,'','15.98',100,'DVD (1)','Kane & Lynch: Dead Man','KiMO','','',100,NULL,'GQ - recrop','Cristan');
```

Comment:
Aici ar fi trebuit `Dead Men`, dar se poate modifica si ulterior.

Reply:
Fixat in #4